packages/storybook/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "lib",
+    "target": "es5",
+    "module": "commonjs",
+    "lib": ["es5", "esnext", "dom"],
```

Comment:
Does this package have an IE 11 support requirement? If so you need to remove the `esnext` lib. (If not, you can remove `es5` since it's automatically included by `esnext`.)